arches_for_science/views/update_resource_list.py

```diff
@@ -57,7 +57,7 @@ def create_new_collection(self, request, transaction_id, project_name):
         tile.data[name_node_id] = stringDataType.transform_value_for_tile("Collection for {0}".format(project_name))
         tile.save(request=request, transaction_id=transaction_id, index=False)
 
-        resource.calculate_descriptors()
+        resource.save_descriptors()
```

Comment:
It looks like we might not need to do this based on this PR: https://github.com/archesproject/arches-for-science/pull/1499/files

Reply:
Yeah, it did occur to me we probably didn't need to resave the resource. I'll back out this change so there isn't a merge conflict later.